pkg/apm/applications_test.go

```diff
@@ -190,3 +274,88 @@ func TestDeleteApplication(t *testing.T) {
 	assert.NotNil(t, actual)
 	assert.Equal(t, &testApplication, actual)
 }
+
+func TestMetricNames(t *testing.T) {
+	t.Parallel()
+	apm := newMockResponse(t, testMetricNamesJson, http.StatusOK)
+
+	actual, err := apm.MetricNames(testApplication.ID, MetricNamesParams{})
+	expected := testMetricNames
+
+	assert.NoError(t, err)
+	assert.NotNil(t, actual)
+	assert.Equal(t, len(expected), len(actual))
+
+	if len(expected) == len(actual) {
+		for i := range expected {
+			assert.Equal(t, expected[i], actual[i])
+		}
+	}
+}
+
+func TestMetricData(t *testing.T) {
+	t.Parallel()
+	apm := newMockResponse(t, testMetricDataJson, http.StatusOK)
+
+	type Values struct {
```

Comment:
If we added this struct to the `apm_types.go` it could be used for the `MetricTimeslice.Values` field as well as here. Thoughts?

Reply:
That sounds good.  I think I had that in my head when I was writing it and then forgot.